src/views/MainScreen.tsx

```diff
@@ -27,39 +31,77 @@ const styles = StyleSheet.create({
   }
 });
 
-export default function MainScreen() {
-  const userId = 1;
-  const [exerciseLabel, exerciseLabelUpdate] = useState('My exercise');
-  const [exerciseCalories, exerciseCaloriesUpdate] = useState('100');
-
+const getTomorrowString = (): string => {
```

Comment:
Could use a method comment to document what a "tomorrow string" is and what we will use it for.